hk_install.sh

```diff
@@ -0,0 +1,81 @@
+#!/bin/bash
+
```

Comment:
Does this script need to expose any ports?    

Also, for the 4GB node I am running, I practically have to have a swap file, with swappiness set to 10.  Do we want to create a swap file for them in this script or is that too intrusive?

Reply:
The iota documentation does say that 4GB is just barely above minimum reqs, so I just assumed were going to say the minimum RAM for a hooknode is 6GB.

We can create a swap file, but it does seem a little too intrusive. And if the node doesn't have a SSD it could get some slowdowns. I would simply up the minimum requirements.